src/wp-includes/class-wp-theme-json.php

```diff
@@ -2446,12 +2447,18 @@ protected static function get_setting_nodes( $theme_json, $selectors = array() )
 	 *     ]
 	 *
 	 * @since 5.8.0
+	 * @since 6.6.0 Added options array for modifying generated nodes.
 	 *
 	 * @param array $theme_json The tree to extract style nodes from.
 	 * @param array $selectors  List of selectors per block.
+	 * @param array $options {
+	 *     Optional. An array of options for now used for internal purposes only (may change without notice).
+	 *
+	 *     @type bool   $block_style_variations  Includes style nodes for block style variations. Default false.
```

Comment:
```suggestion
	 *     @type bool $block_style_variations Includes style nodes for block style variations. Default false.
```
Spaces not needed here.